packages/klighd-core/src/search-bar/searchbar-actions.ts

```diff
@@ -0,0 +1,498 @@
+import { Action, CenterAction, SetModelAction, SModelElement, UpdateModelAction } from 'sprotty-protocol'
+import { ActionHandlerRegistry, IActionDispatcher, SetUIExtensionVisibilityAction, TYPES } from 'sprotty'
+import { SearchBar } from './searchbar'
+import { IActionHandler } from 'sprotty'
+import { SearchBarPanel } from './searchbar-panel'
+import { inject, injectable } from 'inversify'
+import { isContainerRendering, isKText, KColoring, KRectangle, KText } from '../skgraph-models'
+import { rgb } from 'sprotty'
+import { createSemanticFilter } from '../../src/filtering/util'

```

Comment:
This import seems to break the automatic GH Actions build. Its seems be because you go up two folders, just to go back to the source folder here. When built, the src folder is not there anymore, so this import cannot be resolved correctly. Replace this import with `from '../filtering/util'` instead.